environment.yml

```diff
@@ -11,3 +11,4 @@ dependencies:
   - tensorflow=>1.14
   - tensorflow-probability
   - zfit
+  - asdf
```

Comment:
This is a relatively new package. I'm not saying you should not use it, but do you have strong-ish reasons to use it rather than use other packages for the same job?

Reply:
The main motivation is that it stores array in yml files, so when serialisation of likelihood will be developed toys will be saved in the same yml file as the likelihood.

Reply:
Neat! Thanks for the explanation. Good point.